src/geodata2.erl

```diff
@@ -82,15 +82,13 @@ new(ConfigName, Ets) ->
 
 -spec init(_) -> {ok, state()} | {stop, tuple()}.
 init(_Args) ->
-    case new(dbfile, ?GEODATA2_STATE_TID) of
+    Res = load_files(),
+    maybe_launch_reload(),
```

Comment:
🤔 The name is a little misleading since it will not launch anything, I think a better name would be `maybe_schedule_reload`